src/webui/api/synccontroller.cpp

```diff
@@ -521,7 +525,11 @@ void SyncController::torrentPeersAction()
 qint64 SyncController::getFreeDiskSpace()
 {
     if (m_freeDiskSpaceElapsedTimer.hasExpired(FREEDISKSPACE_CHECK_TIMEOUT)) {
-        QTimer::singleShot(0, m_freeDiskSpaceChecker, &FreeDiskSpaceChecker::check);
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 10, 0))
```

Comment:
Maybe extract these lines into separate function to avoid duplication?

Reply:
Done.